themes/default/content/docs/get-started/_index.md

```diff
@@ -15,32 +15,25 @@ aliases:
 
 Pulumi is a modern infrastructure as code platform that allows you to use familiar programming languages and tools to build, deploy, and manage cloud infrastructure.
 
-Select one of the following options to deploy a simple application in your cloud using Pulumi:
+We have guides for AWS, Azure, Google Cloud and Kubernetes using TypeScript, JavaScipt, Python, Go or C#.
 
-<div class="tiles flex-wrap mt-4">
-    <div class="pb-4 md:pr-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "/docs/get-started/aws" >}}">
-            <img class="h-10 mx-auto" src="/logos/tech/aws.svg" alt="AWS">
-        </a>
-    </div>
-    <div class="pb-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "/docs/get-started/kubernetes" >}}">
-            <img class="h-10 mx-auto" src="/logos/tech/k8s.svg" alt="Kubernetes">
-        </a>
-    </div>
-    <div class="pb-4 md:pr-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "/docs/get-started/azure" >}}">
-            <img class="h-10 mx-auto" src="/logos/tech/azure.svg" alt="Azure">
-        </a>
-    </div>
-    <div class="pb-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "/docs/get-started/gcp" >}}">
-            <img class="h-10 mx-auto" src="/logos/tech/gcp.svg" alt="Google Cloud">
-        </a>
-    </div>
-</div>
+## Deploy an Application Guides
+
+- [AWS and Pulumi Guide]({{< relref "/docs/get-started/aws" >}})
+- [Azure and Pulumi Guide]({{< relref "/docs/get-started/azure" >}})
+- [Google Cloud and Pulumi Guide]({{< relref "/docs/get-started/gcp" >}})
+- [Kubernetes and Pulumi Guide]({{< relref "/docs/get-started/kubernetes" >}})
+
+## More Get Started Resources
 
-Or, watch how to do it in this video walkthrough:
+- Learn about [Pulumi Architecture & Concepts]({{< relref "/docs/intro/concepts" >}})
+- Learn how to adopt Pulumi in the [Adopting Pulumi Guide]({{< relref "/docs/guides/adopting" >}})
+- Sign up for one of our [Live Workshops]({{< relref "/resources/introduction-to-pulumi" >}})
+- Learn about all the other clouds we support in the [Cloud Provider Docs]({{< relref "/docs/intro/cloud-providers" >}})
+
+### Demo Video
```

Comment:
I'd personally keep this where it is now, in terms of page hierarchy -- in other words, as the second most important thing on the page to potential new users, ahead of the deeper dives into architecture & concepts, adopting, etc. Unless we have signal we _shouldn't_ do this, it'd make more sense to me to assume that someone new(ish) to this page is looking mainly for high-level things -- a hands-on intro or a demo of one -- and secondarily for lower-level ones like how it works, how to adopt it, and so on.

Reply:
i was struggling with this, because i think the thumbnail is heavy, so that's why I placed it lower. i was also considering only using a link, then it could def be second. ill think about it some more.

Reply:
Yeah. Another thing that gets lost here that I didn't notice at first is that the provider logos actually are pretty strong, visually -- it'd be great to be able to keep those if we can.